sdk/core/azure-core-management/src/main/java/com/azure/core/management/Region.java

```diff
@@ -0,0 +1,350 @@
+    /**
+     * West India (Asia Pacific)
+     */
+    public static final Region INDIA_WEST = new Region("westindia", "West India");
+    /*
+     * Azure Cloud - Middle East and Africa
+     */
+    /**
+     * UAE North (Middle East) (recommended)
+     */
+    public static final Region UAE_NORTH = new Region("uaenorth", "UAE North");
+    /**
+     * UAE Central (Middle East)
+     */
+    public static final Region UAE_CENTRAL = new Region("uaecentral", "UAE Central");
+    /**
+     * South Africa North (Africa) (recommended)
+     */
+    public static final Region SOUTHAFRICA_NORTH = new Region("southafricanorth", "South Africa North");
+    /**
+     * South Africa West (Africa)
+     */
+    public static final Region SOUTHAFRICA_WEST = new Region("southafricawest", "South Africa West");
+    /*
+     * Azure China Cloud
+     */
+    /**
+     * China North
+     */
+    public static final Region CHINA_NORTH = new Region("chinanorth", "China North");
+    /**
+     * China East
+     */
+    public static final Region CHINA_EAST = new Region("chinaeast", "China East");
+    /**
+     * China North 2
+     */
+    public static final Region CHINA_NORTH2 = new Region("chinanorth2", "China North 2");
+    /**
+     * China East 2
+     */
+    public static final Region CHINA_EAST2 = new Region("chinaeast2", "China East 2");
+    /*
+     * Azure German Cloud
+     */
+    /**
+     * Germany Central
+     */
+    public static final Region GERMANY_CENTRAL = new Region("germanycentral", "Germany Central");
+    /**
+     * Germany Northeast
+     */
+    public static final Region GERMANY_NORTHEAST = new Region("germanynortheast", "Germany Northeast");
+    /*
+     * Azure Government Cloud
+     */
+    /**
+     * U.S. government cloud in Virginia.
+     */
+    public static final Region GOV_US_VIRGINIA = new Region("usgovvirginia", "US Gov Virginia");
+
+    /**
+     * U.S. government cloud in Iowa.
+     */
+    public static final Region GOV_US_IOWA = new Region("usgoviowa", "US Gov Iowa");
+
+    /**
+     * U.S. government cloud in Arizona.
+     */
+    public static final Region GOV_US_ARIZONA = new Region("usgovarizona", "US Gov Arizona");
+
+    /**
+     * U.S. government cloud in Texas.
+     */
+    public static final Region GOV_US_TEXAS = new Region("usgovtexas", "US Gov Texas");
+
+    /**
+     * U.S. Department of Defense cloud - East.
+     */
+    public static final Region GOV_US_DOD_EAST = new Region("usdodeast", "US DoD East");
+
+    /**
+     * U.S. Department of Defense cloud - Central.
+     */
+    public static final Region GOV_US_DOD_CENTRAL = new Region("usdodcentral", "US DoD Central");
+
+    private final String name;
+    private final String label;
+
+    /**
+     * @return predefined Azure regions
+     */
+    public static Region[] values() {
+        Collection<Region> valuesCollection = VALUES_BY_NAME.values();
+        return valuesCollection.toArray(new Region[valuesCollection.size()]);
```

Comment:
Why not return the collection itself instead of converting it to array?

Reply:
It is kind of enumerating Enum `public static T[] values()`.

Sure to change to `Collection` (as https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core/src/main/java/com/azure/core/util/ExpandableStringEnum.java#L72-L74).